go/teams/teams.go

```diff
@@ -1079,20 +1081,22 @@ func (t *Team) deleteSubteam(ctx context.Context, ui keybase1.TeamsUiInterface)
 		ForceRepoll: true,
 	})
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to load parent team; you must be an admin of a parent team to delete a subteam: %w", err)
```

Comment:
was confusing otherwise... it'd tell you "You are not a member of the team" referring to the parent team, even if you were in the subteam

Reply:
Ok. This will produce a very long error message in the case you've described. And a weird one if there is a network error. You don't have to spend more time on this, but if you did, you could rewrite the specific error code that means you're not in the parent to "must be an admin of the parent team to delete the subteam"